CONTRIBUTOR-ROLE.md

```diff
@@ -0,0 +1,96 @@
+team has a designated Security Lead. Currently,
+[Tuomo Tanskanen](https://github.com/tuminoid) is acting as the Metal3 Security
+Lead. Below are the responsibilities of the Security team.
+
+* Responsibilities:
+
+   * Thoroughly investigate all reports to the Security Team
+   * Does not share any vulnerability information with others unless necessary
+    to fix the issue
+   * Notify the reporter as the security issue moves through the identification
+    and resolution process
+   * Uphold the [Project Security Policy](https://book.metal3.io/security_policy#metal3-security-committee-members).
+
+* Qualifications:
+
+   * Has a solid grasp of the overall architecture of Metal3, its usage in
+    production environments, and its threat model.
+   * Established the trust of Metal3 community members that they will uphold the
+    security posture of Metal3
+
+* Privileges:
+
+   * Access to security reports
+   * Membership of the security team mailing list
+   * Security Lead has elevated access to Metal3 github organization through
+   the `metal3_security_team` GitHub team.
+
+The process of becoming a security team member is:
+
+1. Nominated by a security team member and seconded by a maintainer
+1. Shadowing an existing security team member
+
+## CI Team
+
+The CI team works to keep the CI infrastructure up and running. Please contact
+CI team at `metal3-ci@googlegroups.com`. (N.B. This group doesn't exist yet).
+
+* Responsibilities:
+
+   * Keeping CI infrastructure healthy
+   * Investigating CI infrastructure problems
+   * Provide visibility into CI spend and status
+
+* Qualifications:
+
+   * Must be a Metal3 Org member
+   * A basic understanding of how the CI system works
+
+* Privileges:
+
+   * Privileged access rights to configuring CI infrastructure is available
+   to a small subset of people currently.
+
+## Release Team
+
+The Release team is responsible for releasing minor and patch releases from
+different repositories in Metal3. Please contact Release team at
+`metal3-release@googlegroups.com`. (N.B. This group doesn't exist yet).
+
+* Responsibilities:
+
+   * Making sure the repositories are ready to be released.
+   * Performing minor and patch releases.
+   * Communicate the releases to the community
+
+* Qualifications:
+
+   * Must be a Metal3 Org member
+   * A proper understanding of release process of different Metal3 repositories.
+
+* Privileges:
+
+   * Release team requires elevated access to Metal3 github organization through
+   the `metal3_release_team` github team.
+
+The process of becoming a release team member is:
+
+1. Nominated by a community member and seconded by a maintainer
+1. Shadowing an existing release team member
+
+## Maintainers
+
+[Maintainers](maintainers/ALL-OWNERS) are very established contributors who are
+responsible for the entire project. Detailed responsibilities and guideline to
+become a maintainer is documented [here](maintainers/README.md).
```

Comment:
Instead of pointing to that readme, it should point to CONTRIBUTOR-LADDER.md

Reply:
actually I wanted to point to README.

Reply:
OK, in that case can we more refer to the README? If we create second copy of the same info, but a bit different. Having it in multiple places and especially having content differ just creates confusion.

Reply:
Yes I can agree to that. I thought of referring the section here but then moved on to put a concise version here to make it consistent with other roles of this document. Do you want me to remove everything I wrote here and just refer to the README?

Reply:
IMO not remove everything, leave the responsibilities and privileges part so it is consistent with other roles, but replace the "becoming approver" part with a link to README, since README is all about becoming approver.